src/bitwarden_exporter/bw_cli.py

```
@@ -63,7 +73,7 @@ def bw_exec(
     Raises:
         ValueError: If the command returns a non-zero exit status.
     """
-    cmd = [BITWARDEN_SETTINGS.bw_executable] + cmd
+    cmd = [settings.bw_executable] + cmd
```

Ruff (0.14.1)

Consider `[settings.bw_executable, *cmd]` instead of concatenation

Replace with `[settings.bw_executable, *cmd]`

(RUF005)